File: backend/src/main/java/cohappy/backend/model/dto/response/UserChatDTO.java
package cohappy.backend.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class UserChatDTO {
    private String chatCode;
    private List<String> participating;
    private String name;
    private byte[] immage;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/response/UserChatDTO.kt
package cohappy.frontend.model.dto.response
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class UserChatDTO(
    var chatCode: String? = null,
    var participating: List<String>? = null,
    var name: String? = null,
    var immage: ByteArray? = null
)